velox/type/TimestampConversion.cpp

```diff
@@ -865,9 +995,15 @@ fromTimestampWithTimezoneString(
       timezonePos++;
     }
 
-    std::string_view timeZoneName(str + pos, timezonePos - pos);
+    std::string timeZoneName(str + pos, timezonePos - pos);
+    std::string normalizeTzName = timeZoneName;
```

Comment:
Ditto. Can we use std::string_view?

Reply:
We can use `std::string_view` for `timeZoneName`, but `normalizeTzName` need to be `std::string`, because we will change its content in `normalizeSparkTimezone`.